requirements/dev.txt

```diff
@@ -182,15 +180,10 @@ pytest==7.2.0
     # via
     #   -r requirements/needle.txt
     #   pytest-cov
-    #   pytest-forked
     #   pytest-xdist
 pytest-cov==4.0.0
     # via -r requirements/needle.txt
-pytest-forked==1.4.0
-    # via
-    #   -r requirements/needle.txt
-    #   pytest-xdist
-pytest-xdist==2.5.0
+pytest-xdist==3.0.2
```

Comment:
Removed the `--boxed` parameter in the new version https://pytest-xdist.readthedocs.io/en/latest/changelog.html#pytest-xdist-3-0-2-2022-10-25